android/src/main/java/com/swmansion/rnscreens/gamma/tabs/TabsHost.kt

```diff
@@ -370,6 +379,40 @@ class TabsHost(
         }
     }
 
+    private fun updateBadgeAppearance(
+        menuItemIndex: Int,
+        tabScreen: TabScreen,
+    ) {
+        if (tabScreen.tabBarItemBadgeVisible != true) {
+            val badge = bottomNavigationView.getBadge(menuItemIndex)
+            badge?.isVisible = false
+
+            return
+        }
+
+        val badgeValue = tabScreen.badgeValue
+        val badgeValueNumber = badgeValue?.toIntOrNull()
+
+        val badge = bottomNavigationView.getOrCreateBadge(menuItemIndex)
+        badge.isVisible = true
+
+        if (badgeValueNumber != null) {
+            badge.number = badgeValueNumber
+        } else if (badgeValue != null) {
+            badge.text = badgeValue
+        } else {
+            badge.clearNumber()
+            badge.clearText()
+        }
```

Comment:
Okay, so badgeValue is either a string or a number. If it is a number I guess we get this nice shortening mechanism (picture below 👇🏻), is that right? 

We need to document this behaviour. Please let's start creating short documentation around exposed types (similar to what we have in v4 implementation). We need to have such interactions described. It can be done in separate PR, I don't mind here. 

<img width="100" height="76" alt="image" src="https://github.com/user-attachments/assets/c510b69a-c525-4782-91aa-180a8bbbfaff" />